### PR TITLE
[WIP] command line arguments update

### PR DESCRIPTION
**Pull request by sedflix**

### CODE REVIEW

File: src/main/java/fr/inria/coming/main/ComingMain.java [hunk 1]
```diff
@@ -59,35 +59,69 @@
...
+        // Revision filter
+        options.addOption("filter", true, "name of the filter");
+        options.addOption("filtervalue", true, "values of the filter  mentioned in -filter");
+
+        // In case of git
+        options.addOption(Option.builder("branch")
+                .argName("branch name").hasArg()
+                .desc("In case of -input=\'git\', use this branch name. Default is master.")
+                .build());
+        options.addOption("message", true, "comming message"); // TODO: What's this?
```
Comment (sedflix) [inline src/main/java/fr/inria/coming/main/ComingMain.java]: What is the purpose of this argument?

Comment (martinezmatias) [reply src/main/java/fr/inria/coming/main/ComingMain.java]: > What is the purpose of this argument?

Using this argument you can specify the branch to analyze. By default Coming analyzes the branch Master

**Comment by martinezmatias** [comment]
HI @sedflix 
Thanks for the PR.
I observe that it updates the comments of each argument. Did you add/remove any argument?

File: src/main/java/fr/inria/coming/main/ComingMain.java (hunk 1 above)
Comment (sedflix) [reply src/main/java/fr/inria/coming/main/ComingMain.java]: I was talking about the message argument.

**Comment by sedflix** [comment]
Nope, I've not removed or added any.

**Comment by martinezmatias** [comment]
Okey, perfect

**Comment by martinezmatias** [comment]
Merged. Thanks @sedflix

**Pull request merged**